src/components/ArticleDetail/Vote/Vote.jsx

```diff
@@ -0,0 +1,44 @@
+import './Vote.css'
+import React, { useState } from 'react';
+import { updateArticleVotes } from '../../../utils/api'; 
+
+
+const Vote = ({ articleId, currentVotes }) => {
+    const [votes, setVotes] = useState(currentVotes); 
+    const [buttonClicked, setButtonClicked] = useState(false); 
+
+    const updateVotes = async (voteChange) => {
+        try {
+            setButtonClicked(true)
```

Comment:
the first time you vote, you are setting button clicked to true, then when you vote again, as button clicked is true, this function is not invoked and therefore you cannot vote again until you refresh the page which refreshes the state back to false and allows this function to be invoked again